src/common/config/defaultPreferences.ts

```diff
@@ -12,12 +12,14 @@ import os from 'os';
 
 import {ConfigV3} from 'types/config';
 
-export const getDefaultDownloadLocation = (): string => {
+export const getDefaultDownloadLocation = (): string | undefined => {
     // eslint-disable-next-line no-undef
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore
-    const homePath = __IS_MAC_APP_STORE__ ? path.join(os.homedir(), '../../../../') : os.homedir();
-    return path.join(homePath, 'Downloads');
+    if (__IS_MAC_APP_STORE__) {
+        return undefined;
```

Comment:
0/5, but would it help anything if we used the empty string instead of undefined here? I'm still on a mission to get JS to work like Go where the zero value for a string is the empty string instead of undefined 😛